acme/.pylintrc

```diff
@@ -60,7 +60,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used
```

Comment:
I'll roll this back with the Travis config rollback. Its relevance lay in the Travis config, in that the linter mysteriously ignored the too-many-attrs "problem" on master but not on this branch.